docs/config/configuring-nodes-storage.rst

```diff
@@ -31,12 +31,15 @@ The optional ``port`` variable is the port on the control node to connect to.
 This value must agree with the configuration for the control service telling it on what port to listen.
 Omit the ``port`` from both configurations and the services will automatically agree.
 
+If node has a public address that is different than its local address (for example, on an EC2 instance),
```

Comment:
This isn't strictly necessary though, right? We after all manage without right now. And it adds an extra burden of configuration on users that they don't necessarily need. Maybe indicate it's useful but not required? Or the caveats of not doing it?